internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -153,30 +156,45 @@ func parseSetStatement(statement string) (string, string, error) {
 	}
 
 	if !strings.Contains(strAfterSet, "=") {
-		return "", "", &types.StatementError{Msg: "Error: missing \"=\". Usage example: SET 'key'='value'."}
+		return "", "", &types.StatementError{
+			Msg:   `Missing "="`,
+			Usage: []string{"SET 'key'='value'"},
+		}
 	}
 
 	keyValuePair := strings.Split(strAfterSet, "=")
 
 	if len(keyValuePair) != 2 {
-		return "", "", &types.StatementError{Msg: "Error: \"=\" should only appear once. Usage example: SET 'key'='value'."}
+		return "", "", &types.StatementError{
+			Msg:   `"=" should only appear once`,
+			Usage: []string{"SET 'key'='value'"},
+		}
 	}
 
 	if keyValuePair[0] != "" && keyValuePair[1] == "" {
-		return "", "", &types.StatementError{Msg: "Error: Value for key not present. If you want to reset a key, use \"RESET 'key'\"."}
+		return "", "", &types.StatementError{Msg: `Value for key not present. If you want to reset a key, use "RESET 'key'"`}
```

Comment:
Looks like the second sentence belongs in `Usage`

Reply:
Moved it to new Suggestion field